lib/folio_client/token_wrapper.rb

```diff
@@ -4,7 +4,7 @@ class FolioClient
   # Wraps API operations to request new access token if expired
   class TokenWrapper
     def self.refresh(config, connection)
-      yield.tap { |response| UnexpectedResponse.call(response) unless response.success? }
```

Comment:
As a result of the refactoring, we can drop this bit which always twisted my wig for some reason.